src/facade/dragonfly_connection.cc

```diff
@@ -567,7 +567,12 @@ void Connection::OnPostMigrateThread() {
     socket_->RegisterOnErrorCb([this](int32_t mask) { this->OnBreakCb(mask); });
   }
 
-  DCHECK(!dispatch_fb_.IsJoinable());
+  // If someone had sent Async during the migration, dispatch_fb_ will be created.
+  if (dispatch_fb_.IsJoinable()) {
+    // How can we ensure that dispatch_fb_ is created on the correct thread?
+    // TODO: to introduce Fiber::IsLocal method.
+    CHECK(!dispatch_q_.empty());
```

Comment:
Just making sure: did you intentionally change from `DCHECK` to `CHECK`?

Reply:
you are right, downgraded.